lib/express/server.js

```diff
@@ -23,7 +23,7 @@ import { DEFAULT_BASE_PATH } from '../constants';
 import { EventEmitter } from 'events';
 
 
-const KEEP_ALIVE_TIMEOUT_MS = 60 * 10 * 1000; // 10 minutes
+const KEEP_ALIVE_TIMEOUT_MS = 60 * 60 * 1000; // 60 minutes
```

Comment:
Is it better to leave a comment we should keep this timeout (not to make this short in future change)?

Reply:
yeah probably. we might also eventually want to make it configurable via a cli arg if a long timeout is bad for some people